tests/Unish/UnishTestCase.php

```diff
@@ -276,12 +281,58 @@ function setUpDrupal($num_sites = 1, $install = FALSE, $version_string = UNISH_D
     return self::$sites;
   }
 
-  function fetchInstallDrupal($env = 'dev', $install = FALSE, $version_string = UNISH_DRUPAL_MAJOR_VERSION, $profile = NULL) {
-    $options = array();
-    $root = $this->webroot();
-    $site = "$root/sites/$env";
+  function setUpSingleSiteDrupal($env = 'dev', $install = FALSE, $version_string = UNISH_DRUPAL_MAJOR_VERSION, $profile = NULL) {
```

Comment:
This looks like a lot of duplicated code. Did you consider my proposal to use archive-dump/restore to get a 2nd site? Would also save on awkward sql-sync and uuid fiddling during test.